airesources/Rust/src/hlt/game.rs

```diff
@@ -41,27 +40,29 @@ impl Game {
 
         println!("{}", name);
```

Comment:
Would you mind attempting to address https://github.com/HaliteChallenge/Halite-II/issues/151 at the same time?

Reply:
I will fix that yes, thanks for the remark.